pages/common/assimp.md

```diff
@@ -0,0 +1,28 @@
+# assimp
+
+> Command-line client for the Open Asset Import Library.
+> Supports loading of 40+ 3D file formats, and exporting to several popular 3D formats.
+
+- List all supported import formats:
+
+`assimp listext`
+
+- List all supported export formats:
+
+`assimp listexport`
+
+- Convert a file to one of the supported output formats (see dox_cmd.h for a list of all common parameters):
```

Comment:
Just checking - the user seeing this page will know what does dox_cmd.h refer to right ?